src/inc/msquic_linux.h

```diff
@@ -20,6 +20,7 @@
 #define _MSQUIC_LINUX_
 
 #include <stdlib.h>
+#include <stdio.h>
```

Comment:
Absolutely necessary?

Reply:
Yes. That's where sprintf is pulled in. We've just gotten lucky in the past that any time we've included this and used the sprintf_s macro, its been included. But I ran into a case here where I wasn't including it, and we should be including what we use.